python/processors/README.md

```diff
@@ -20,3 +20,12 @@ This example demonstrates the simplest possible variant of a processor. There ar
 It also demonstrates a parser function that outputs operations to create, update, and delete rows.
 
 This processor is part of a full Aptos project, so you can see the processor in context alongside the Move module and frontend.
+
+## Aptos Ambassador Token
```

Comment:
Can you add some instructions on how to run the processor code?

Reply:
Added.